apps/expo/src/app/_layout.tsx

```diff
@@ -31,8 +32,13 @@ const RootLayout = () => {
         The Stack component displays the current page.
         It also allows you to configure your screens 
       */}
-          <NavBar />
-          <StatusBar />
+          <SignedOut>
+            <SignInScreen />
+          </SignedOut>
+          <SignedIn>
+            <NavBar />
+            <StatusBar />
+          </SignedIn>
```

Comment:
can you revert this logic until we get working passwordless, so that devs don't need to worry about creating an account